L1Trigger/L1TTrackMatch/plugins/L1TrackJetEmulationProducer.cc

```diff
@@ -243,6 +244,10 @@ void L1TrackJetEmulationProducer::produce(Event &iEvent, const EventSetup &iSetu
     delete[] mzb.clusters;
   } else if (L1TrkPtrs_.empty()) {
```

Comment:
Hello, yes it is just the else.

Reply:
Should we change to `else` then?

Reply:
Sorry, I confuse. Will you rewrite the following PR? Then I am ok.

Reply:
Hello,

I am re-writing the code altogether. I can make this change in this PR or I can leave it as is for now, and then when l1t offline gets merged it will change automatically. Sorry for the confusion.

Best,
george

Reply:
OK thanks. Then I think we can leave this for now, when PR test is done and success, we can sign.